lang/cs/Org.Apache.REEF.IMRU/API/IIMRUCheckpointResultHandler.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using Org.Apache.REEF.IMRU.OnREEF.CheckpointHandler;
+using Org.Apache.REEF.Tang.Annotations;
+
+namespace Org.Apache.REEF.IMRU.API
+{
+    /// <summary>
+    /// Internal interface that defineds methods for checkpoint.
+    /// </summary>
+    [DefaultImplementation(typeof(IMRUCheckpointResultHandler))]
+    internal interface IIMRUCheckpointResultHandler
+    {
+        /// <summary>
+        /// Set a flag to indicate the result has been handled.
+        /// </summary>
+        void SetResult();
+
+        /// <summary>
+        /// Check if the result flag has been set.
+        /// </summary>
+        bool GetResult();
```

Comment:
This needs a better name, e.g. `IsResult`?

Reply:
done